task/github-request-reviewers/0.1/github-request-reviewers.yaml

```
@@ -0,0 +1,113 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: github-request-reviewers
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.17.0"
+    tekton.dev/tags: github
+    tekton.dev/displayName: "request github reviewers"
+spec:
+  description: >-
+    This Task will request reviewers for a GitHub PR
+
+    It can take either a list of users or teams or both.
+    Note that, when teams are used, GitHub does not verify
+    if the team (or any of its members) are collaborators
+    the repo, so a 201 reply may still result in no reviewer
+    added to the PR.
+  workspaces:
+    - name: github
+      description: The workspace that contains the GitHub secret
+  params:
+    - name: GITHUB_HOST_URL
+      description: |
+        The GitHub host, adjust this if you run a GitHub Enteprise.
+      default: "api.github.com"
+      type: string
+
+    - name: API_PATH_PREFIX
+      description: |
+        The API path prefix, GitHub Enterprise has a prefix e.g. /api/v3
+      default: ""
+      type: string
+
+    - name: PACKAGE
+      description: "{owner}/{repo}"
+      type: string
+
+    - name: PULL_REQUEST_NUMBER
+      description: The pull request number
+      type: string
+
+    - name: REVIEWERS
+      description: |
+        Command separated list of github user slugs.
```

```suggestion
        Comma separated list of github user slugs.
```
Maybe this was supposed to be comma?

---

Nice catch, thank you!